bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tscatter.py

```diff
@@ -0,0 +1,73 @@
+r'''
```

Comment:
Can you remove this file? It's already being addressed by https://github.com/root-project/root/pull/17039

Reply:
The version in this PR is more recent. It uses the new tags. I guess the other PR can be closed.

Reply:
The other PR is a user contribution so I would rather merge that one, then update this.